src/shared/config.py

```diff
@@ -32,3 +36,13 @@
                                 APIS_PATH,
                                 SEC_FOLDER,
                                 SEC_SCHEMAS_FOLDER)
+
+
```

Comment:
I think we could do something like that instead of creating a function and variable after it: 
ENV_CONFIG = {
        **dotenv_values(f"{PROJECT_PATH}/.env.secrets"),  # load sensitive variables
        **os.environ,  # override loaded values with environment variables
    }

What do you think?

Reply:
I don't understand??? Check the method load_dot_env()